upgrade/upgrade-instructions-for-improved-data-fixtures-and-tests.md

```diff
@@ -0,0 +1,63 @@
+        ```
+    - Use [`PriceConverter`](https://github.com/shopsys/shopsys/blob/v8.1.0/packages/framework/src/Model/Pricing/PriceConverter.php) class for converting prices into all currencies, e.g.:
+        ```php
+          foreach ($this->domain->getAllIncludingDomainConfigsWithoutDataCreated() as $domain) {
+              $currency = $this->currencyFacade->getDomainDefaultCurrencyByDomainId($domain->getId());
+              $price = $this->priceConverter->convertPriceWithoutVatToPriceInDomainDefaultCurrency($price, $domain->getId());
+  
+              $transportData->pricesByCurrencyId[$currency->getId()] = $price;
+          }
+        ``` 
+- Modify your functional tests:
+    - Add `skipTestIfFirstDomainIsNotInEnglish`, `getReferenceForDomain`, and `getFirstDomainLocale` methods to your [`FunctionalTestCase`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/tests/ShopBundle/Test/FunctionalTestCase.php).
+    - When accessing a translated attribute on first domain, use `FunctionalTestCase::getFirstDomainLocale` instead of hard-coding the locale.
+    - In tests that make sense only for the English locale (e.g. tests for searching a particular phrase), you can skip their execution for different settings using `FunctionalTestCase::skipTestIfFirstDomainIsNotInEnglish`.
+    - When asserting prices, use [`PriceConverter`](https://github.com/shopsys/shopsys/blob/v8.1.0/packages/framework/src/Model/Pricing/PriceConverter.php) class to convert the prices to proper currency.
+    - Use `ParameterFacade::getParameterValueByValueTextAndLocale()` to get the proper entity in tests where the `ParameterValue` ids are used.
+    - Modify your `app/config/packages/test/prezent_doctrine_translatable.yml` to use `%locale%` value for setting `fallback_locale` instead of hard-coding it.
+- Modify your acceptance tests:
+    - Add two new modules - [`LocalizationHelper`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/tests/ShopBundle/Test/Codeception/Helper/LocalizationHelper.php) and [`NumberFormatHelper`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/tests/ShopBundle/Test/Codeception/Helper/NumberFormatHelper.php)
+        - The modules add new methods to acceptance tester that are useful in the tests, e.g.:
+            - `amOnLocalizedRoute()`
+            - `seeTranslationAdmin()`
+            - `seeTranslationFrontend()`
+            - `getFormattedPriceWithCurrencySymbolOnFrontend()`
+        - Rewrite your tests to make them resistant against domains and locales settings using these methods.
+        - Register the new modules in your `acceptance.suite.yml` configuration in `modules.enabled` section.
+    - Update your [`Cart/index.html.twig`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Cart/index.html.twig) - add JS class `js-cart-item-price` it is used for identification of item price in cart in acceptance tests.
```

Comment:
add the class where?

Reply:
fixed in 3e53ddd53b6421b5025579dee4a29f0dcefefa85